src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,105 +1,49 @@
-  private Joystick joystick = new Joystick(RobotMap.JOYSTICK_PORT1);
-
-  private final DrivetrainSubsystem drivetrainSubsystem = new DrivetrainSubsystem();
-  private final PowerDistribution powerDistribution = new PowerDistribution();
+    private Joystick joystick = new Joystick(Constants.JOYSTICK_PORT1);
+    private Joystick armJoystick = new Joystick(Constants.JOYSTICK_PORT2);
 
-  //auto stuff
-  private String path = "Pathweaver/paths/exit1.path";
-  private AutoUtils autoUtils = new AutoUtils(drivetrainSubsystem);
+    private final Drivetrain drivetrain = new Drivetrain();
+    private final ArmSubsystem arm = new ArmSubsystem();
 
-
-
-  public RobotContainer() {
-    //register subsystems and set default commands
-    powerDistribution.clearStickyFaults();
+    private final PowerDistribution powerDistribution = new PowerDistribution();
+    private final AutoUtils autoUtils = new AutoUtils();
     
-    CommandScheduler.getInstance().registerSubsystem(drivetrainSubsystem);
-    drivetrainSubsystem.setDefaultCommand(new DriveCommand(drivetrainSubsystem,
-      () -> -joystick.getRawAxis(0),
-      () -> -joystick.getRawAxis(1)));
-
-    configureButtonBindings();
-  }
-
-
-  private void configureButtonBindings() {
-
-  }
+    public RobotContainer() {
+        powerDistribution.clearStickyFaults();
 
+        startDefaultCommands();
 
-  public DrivetrainSubsystem getDrive() {
-    return drivetrainSubsystem;
-  }
+    }
 
-  public SequentialCommandGroup simpleRamseteConfig() {
-        TrajectoryConfig config =
-        new TrajectoryConfig(
-                Constants.DriveConstants.Autos.maxVelocity,
-                Constants.DriveConstants.Autos.maxAccel)
-            .setKinematics(Constants.DriveConstants.DRIVE_KINEMATICS);
+    public void startDefaultCommands() {
+        drivetrain.setDefaultCommand(new SimpleDriveCommand(drivetrain, 
+        () -> -joystick.getRawAxis(1),
+        () -> -joystick.getRawAxis(0)));
 
-      Trajectory traj = TrajectoryGenerator.generateTrajectory(
-            new Pose2d(0, 0, new Rotation2d(0)),
-            List.of(new Translation2d(1, 1), new Translation2d(2, -1)),
-            new Pose2d(3, 0, new Rotation2d(0)),
-            config);
-       RamseteCommand ramseteCommand =
-        new RamseteCommand(
-            traj,
-            drivetrainSubsystem::getPose,
-            new RamseteController(),
-            new SimpleMotorFeedforward(
-            Constants.DriveConstants.Autos.kS, 
-            Constants.DriveConstants.Autos.kV, 
-            Constants.DriveConstants.Autos.kA),
-            Constants.DriveConstants.DRIVE_KINEMATICS,
-            drivetrainSubsystem::getWheelSpeeds,
-            new PIDController(0.1, 0, 0),
-            new PIDController(0.1, 0, 0),
-            drivetrainSubsystem::tankDriveVolts,
-            drivetrainSubsystem);
+        arm.setDefaultCommand(new RunCommand(() -> arm.moveArm(() -> -armJoystick.getRawAxis(0)), arm));
```

Comment:
Is forward on the joystick axis negative or positive? I'm asking because I see you have negated the joystick axis here, so I'm just wondering if "forward" on the joystick translates "up" or "down" on the arm.

Reply:
from my testing, forward translates to down